.settings/.jsdtscope

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Julia, should these .settings files be checked in?  isn't this specific to your dev environment?

Reply:
No, they shouldn't be checked in.  I will remove them.